gen/model/CloudFormationResourceSpecification.json

```diff
@@ -20567,6 +20567,18 @@
         }
       }
     },
+    "AWS::CodeBuild::Project.FilterGroup": {
+      "Documentation": "http://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-properties-codebuild-project-filtergroup.html",
+      "Properties": {
+        "FilterGroups": {
```

Comment:
Hmm I actually think this may be invalid on further review.  As filtergroups is a list of filtergroup which is in turn a list of webhookfilter.  So i think this has to be a list of lists here.

Reply:
hmm i tried some other options and those just remove the other models.  I didn't see any other current examples of a list of a list so may require some additional code in the specifications parising.